doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -386,22 +387,16 @@ It is recommended to follow the “url” link values,
 applicable to retrieve resources. Instead of constructing your own URLs,
 to keep your client decoupled from implementation details.
 
-+ Attributes(object)
-    + entities_url (required, string) - URL which points to the entities resource
-    + types_url (required, string) - URL which points to the types resource
-    + subscriptions_url (required, string) - URL which points to the subscriptions resource
-    + registrations_url (required, string) - URL which points to the registrations resource
 
 + Response 200 (application/json)
 
-        {
-            "entities_url":      "/v2/entities",
-            "types_url":         "/v2/types",
-            "subscriptions_url": "/v2/subscriptions",
-            "registrations_url": "/v2/registrations"
-        }
+    + Attributes (object)
+        + entities_url: /v2/entities (required, string) - URL which points to the entities resource
```

Comment:
Appart from the change from "*" to "+", a "/v2/entities" token has been introduced in the paragraph. I understand that thas is required so Apiary can render the JSON without explicitly including the JSON fragment in the .apib. Is that correct?